modules/LocationUtils.js

```diff
@@ -24,11 +24,58 @@ export const createRouterLocation = (input, parseQueryString, stringifyQuery) =>
   }
 }
 
-export const createRouterPath = (input, stringifyQuery) => {
-  return typeof input === 'string' ? input : createPath({
+export const createRouterPath = (input, stringifyQuery, base) => {
```

Comment:
Why not make this 3rd arg optional, with a default to `''`? Less code changes elsewhere that way.